drivers/sensor/htu21d/Kconfig

```diff
@@ -0,0 +1,21 @@
+# HTU21D temperature & humidity sensor configuration option
+#
+# Copyright (c) 2022 Linumiz
+# Author: Arunmani <arunmani@linumiz.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+config HTU21D
+	bool "Enable htu21d sensor"
+	depends on I2C
+	help
+	 Enable driver for HTU21D sensor.
+
+if HTU21D
+
+config HTU21D_NH_MASTER
+	bool "Allow no hold master mode"
```

Comment:
default !!

Reply:
Default is hold master mode

Reply:
If htu21d configuration is enabled. hold master mode is also enabled in default.
otherwise we should write config HTU21D_N_MASTER like these?